uint32 id
string name
string color
float64 confidence
float64 distance
float64 thetaleft
float64 thetaright
float64[] corners
